content/en/docs/languages/java/jmx.md

```diff
@@ -0,0 +1,281 @@
+(Java Management Extensions) MBeans and export them to OpenTelemetry.
+
+## Overview
+
+JMX (Java Management Extensions) is a Java technology that provides tools for
+managing and monitoring applications, system objects, devices, and
+service-oriented networks. JMX MBeans (Managed Beans) expose management
+attributes and operations that can be monitored.
+
+The OpenTelemetry JMX Metric Insight module allows you to bridge JMX metrics
+into OpenTelemetry, enabling you to:
+
+- Monitor JVM metrics (memory, garbage collection, threads, etc.)
+- Collect metrics from application-specific MBeans
+- Export JMX data alongside other OpenTelemetry telemetry signals
+- Use predefined metric mappings for popular frameworks
+
+## Installation
+
+### Using the Java Agent
+
+The easiest way to collect JMX metrics is using the OpenTelemetry Java Agent
+with the JMX metrics extension:
+
+1. Download the OpenTelemetry Java Agent (if not already installed):
+
+   ```sh
+   curl -L -O https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/latest/download/opentelemetry-javaagent.jar
+   ```
+
+2. Run your application with the agent and enable JMX metrics:
+
+   ```sh
+   java -javaagent:opentelemetry-javaagent.jar \
+     -Dotel.jmx.enabled=true \
+     -jar myapp.jar
+   ```
+
+### As a Standalone Library
+
+You can also use the JMX metrics library as a standalone dependency:
+
+Add the following dependency to your project:
+
+**Maven (`pom.xml`):**
+
+```xml
+<dependency>
+  <groupId>io.opentelemetry.instrumentation</groupId>
+  <artifactId>opentelemetry-jmx-metrics</artifactId>
+  <version>{{% param vers.instrumentation %}}-alpha</version>
+</dependency>
+```
+
+**Gradle (`build.gradle`):**
+
+```groovy
+dependencies {
+  implementation("io.opentelemetry.instrumentation:opentelemetry-jmx-metrics:{{% param vers.instrumentation %}}-alpha")
+}
+```
+
+## Configuration
+
+### Basic Configuration
+
+The JMX metrics collection can be configured using system properties or
+environment variables:
+
+| System Property                | Environment Variable            | Description                                      | Default |
+| ------------------------------ | ------------------------------- | ------------------------------------------------ | ------- |
+| `otel.jmx.enabled`             | `OTEL_JMX_ENABLED`              | Enable JMX metric collection                     | `false` |
+| `otel.jmx.target.system`       | `OTEL_JMX_TARGET_SYSTEM`        | Predefined metric set to use                     | none    |
+| `otel.jmx.interval.milliseconds` | `OTEL_JMX_INTERVAL_MILLISECONDS` | Collection interval in milliseconds              | `10000` |
+| `otel.jmx.service.url`         | `OTEL_JMX_SERVICE_URL`          | JMX service URL for remote connections           | none    |
+| `otel.jmx.groovy.script`       | `OTEL_JMX_GROOVY_SCRIPT`        | Path to custom Groovy script for metric mapping  | none    |
```

Comment:
I don't have a clear recommendation for this yet, but I think that having separate config sections for the agent mode vs. scraper mode might be a better fit and easier to follow. At first glance this feels a bit "blended".